src/carma-platform/docs/guidance_scripts.md

```diff
@@ -229,6 +229,22 @@ If directories are provided:
   - "guidance_steering_analysis.png"
   - "steering_wheel_analysis.png"
 
+### get_planner_trajectory_intervals
+Extract time intervals when a specific planner was active based on trajectory plans.
+Uses topic /guidance/plan_trajectory
```

Comment:
There are a few prior topics that should also be reformatted. Additionally it would be nice to have a list of topics needed for each function. Some of them have this mentioned in their description but  `run_turn_accuracy_analysis` and `run_acceleration_comfort_analysis` are missing topics.

Reply:
Added the missing topics and reformatted